content/200-concepts/100-components/02-prisma-client/053-client-extensions/100-model.mdx

```diff
@@ -138,13 +147,18 @@ You can get the name of the current model at runtime with `Prisma.getExtensionCo
 For example:
 
 ```ts
-// `ctx` refers to the current model
-const ctx = Prisma.getExtensionContext(this)
+// `context` refers to the current model
+const context = Prisma.getExtensionContext(this)
+
+// `context.name` returns the name of the current model
+console.log(context.name)
 
-// `ctx.name` returns the name of the current model
-console.log(ctx.name)
+// Usage
+await(context as any).findFirst({ args })
```

Comment:
this looks wrong to me

Reply:
Are you referring to the type-casting or whether the example snippet would work?

If it's related to type-casting, Serhii created an issue a few days ago: https://github.com/prisma/prisma/issues/20786

The snippet is functional and we use it in the [`exists-fn`](https://github.com/prisma/prisma-client-extensions/blob/4b7fe4549121a8d1eb73499230f84dd1b91733ba/exists-fn/script.ts#L11) example extension.

Reply:
Just looked wrong on a first skim, so 🤷 If it works, resolve and ignore.